src/components/editor/Editor.tsx

```diff
@@ -122,6 +123,54 @@ export class ErxesEditor extends React.Component<ErxesEditorProps> {
       this.props.handleFileInput(e);
     }
   };
+  _handleBeforeInput = () => {
+    const { editorState, integrationKind } = this.props;
+
+    if(integrationKind !== "telnyx"){
+      return 'un-handled';
+    }
+
+    const MAX_LENGTH = 160;
```

Comment:
this 3 lines of code is duplicated multiple times, create one function & call it